covariance/covarianceXsec.h

```diff
@@ -237,6 +264,16 @@ class covarianceXsec : public covarianceBase {
     std::vector<std::string> FarFuncParsNames;
     std::vector<int> FarFuncParsIndex;
 
+  private:
+	//std::vector<std::string> _fNames;
+	std::vector<std::string> _fFancyNames;
+
+	//ETA - do we need these now?
+	std::vector<std::vector<int>> _fNormModes;
+	std::vector<std::string> _fNDSplineNames;
+	std::vector<std::string> _fFDSplineNames;
+	std::vector<std::vector<int>> _fFDSplineModes;
+	int _nNormPars;
```

Comment:
In long term I agree we don't need separate ND and FD spline names

Reply:
yep, this just needs to be fixed in the spline generation tools to be consistent.